spring-cloud-dataflow-docs/src/main/asciidoc/configuration-cloudfoundry.adoc

```diff
@@ -926,7 +928,7 @@ For scheduling, you must add (or update) the following environment variables in
 
 * Enable scheduling for Spring Cloud Data Flow by setting `spring.cloud.dataflow.features.schedules-enabled` to `true`.
 * Bind the task deployer to your instance of PCF-Scheduler by adding the PCF-Scheduler service name to the `SPRING_CLOUD_DATAFLOW_TASK_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[default]_DEPLOYMENT_SERVICES` environment variable.
-* Establish the URL to the PCF-Scheduler by setting the `SPRING_CLOUD_SCHEDULER_CLOUDFOUNDRY_SCHEDULER_URL` environment variable.
+* Establish the URL to the PCF-Scheduler by setting the `SPRING_CLOUD_DATAFLOW_TASK_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[default]_SCHEDULER_PROPERTIES_SCHEDULER_URL` environment variable.
```

Comment:
I now realise that property key should be `scheduler`, not `schedulerProperties`. That's what PR https://github.com/spring-cloud/spring-cloud-dataflow/pull/3977 had as an example while even ENV type used wrong format.

Reply:
I created #4048 for it.

Reply:
This was in fact my mistake as I suggested the changes when the original PR was submitted :-)